charts/mariadb/v7.3.14/questions.yml

```diff
@@ -23,7 +23,7 @@ questions:
     type: string
     label: MariaDB Image Name
   - variable: image.tag
-    default: "10.3.17-debian-9-r0"
+    default: "10.3.22-debian-10-r44"
```

Comment:
you also need to update the repository to `bitnami/mariadb` if u want to use the origin repo.

Reply:
Updated